providers/squadcast/squadcast_provider.go

```diff
@@ -61,6 +64,15 @@ func (p *SquadcastProvider) Init(args []string) error {
 		p.GetTeamID()
 	}
 
+	if args[3] != "" {
```

Comment:
is it mandatory field to be passed?

Reply:
args[3] is the `--service-name` flag
![image](https://user-images.githubusercontent.com/72292326/188503029-3abbd384-2168-49d8-8635-54509254a74c.png)
it is only required for automation rules like  `tagging_rules`, `routing_rules`, `deduplication_rules` and `suppression_rules`

Reply:
ok, if user is importing something else. forcing them to pass this might not be a great idea

Reply:
Yeah they won't have to provide service name for other resources
![image](https://user-images.githubusercontent.com/72292326/188574289-02c31d41-c7a2-4475-af93-d2ae91a3900d.png)
like region, we're not throwing an error in squadcast_provider.go if service name is not provided.

however if they won't provide it for automation rules, then that error is beign handled in their respective files
![image](https://user-images.githubusercontent.com/72292326/188574588-b7ff14e2-7639-4a82-9c02-205020402143.png)

Reply:
sure, thanks